packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubTaskPopup/SubTaskPopupContainer.js

```diff
@@ -3,27 +3,10 @@ import PropTypes from 'prop-types'
 import { observer, MobXProviderContext } from 'mobx-react'
 import SubTaskPopup from './SubTaskPopup'
 
-function useStores() {
-  const stores = React.useContext(MobXProviderContext)
-  const {
-    activeStepTasks
-  } = stores.classifierStore.workflowSteps
-
-  const [activeInteractionTask = {}] = activeStepTasks.filter(task => task.type === 'drawing' || task.type === 'transcription')
-  const {
-    activeMark
-  } = activeInteractionTask
-  return { activeMark }
-}
-
-export default observer(function SubTaskPopupContainer (props) {
-  const {
-    activeMark
-  } = useStores()
-  const {
-    onDelete
-  } = props
-
```

Comment:
[minor/trivial][non-blocking] If the "connect to store" logic has been removed from SubTaskPopupContainer, do we still need the Container to contain the SubTask component?

Reply:
The container also has conditional logic to render the popup. Changing that would break the tests. Technically speaking, a refactor doesn't change tested behaviour, so I left that alone. It could be moved into the `SubTaskPopup` component.

Reply:
It might be worth doing that when the tests are rewritten to remove Enzyme.